base/applications/clipbrd/clipbrd.c

```diff
@@ -690,16 +689,16 @@ int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPWSTR lpCmdLi
             break;
     }
 
-    ZeroMemory(&Globals, sizeof(Globals));
-    Globals.hInstance = hInstance;
+    InitGlobals(hInstance);
 
     ZeroMemory(&wndclass, sizeof(wndclass));
     wndclass.cbSize = sizeof(wndclass);
+    wndclass.style = CS_HREDRAW | CS_VREDRAW;
     wndclass.lpfnWndProc = MainWndProc;
     wndclass.hInstance = hInstance;
     wndclass.hIcon = LoadIconW(hInstance, MAKEINTRESOURCEW(CLIPBRD_ICON));
     wndclass.hCursor = LoadCursorW(0, IDC_ARROW);
-    wndclass.hbrBackground = (HBRUSH)(COLOR_WINDOW + 1);
+    wndclass.hbrBackground = NULL; /* Background will be erased in WM_PAINT handling */
```

Comment:
Well.... Check OnPaint() function. It can return early without drawing anything in some cases (e.g. OpenClipboard failed, or the clipboard is empty, etc.)

Reply:
Reverted.

Reply:
`OnPaint` is fixed.

Reply:
The flickering problem will be fixed in the future PR.